plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -281,6 +308,32 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->resource = flb_sds_create(FLB_SDS_RESOURCE_TYPE);
     }
 
+    flb_kv_init(&ctx->resource_labels);
+
+    /*
+     * Load monitored resource labels from config files.
+     * Labels that don't match for a particular resource type will be
+     * rejected by the API.
+     *
+     * For a list of valid labels, see:
```

Comment:
[Optional] `For a list of valid labels per resource`…

Reply:
Done.